### DISCUSSION TITLE
Star trails mode and duration/start/stop

### DISCUSSION THREAD

**Discussion by bthitney** [gopro/labs#108] [Q&A] [answered]
I was hoping to configure star trails to start at a certain time and end at a certain time, but it isn’t working and not sure if it isn’t possible or if I’m doing it wrong. When I specify star trails mode and a start time, the camera reads the QR and seems to select Star Trails mode and a start time correctly, but then turns off and back on continuously until the start time and just records in video mode. Any tips?

**Comment by dnewman-gpsw** [comment, marked as answer]
You can set this up two (or more) ways, through the camera's menus, or several ways through Labs.  Through Labs your start time would be something like **!23:00N** to sleep boot the camera at 11pm.  The **mST** to switch the mode to Star Trails.  **!S** to Start and !xE to end x seconds later (3 hours = 10800 seconds.)  You don't need the end, just let the battery run out.  So the command could be **!23:00NmST!S**

I believe you tried setting the mode then the wake time.  If you don't make the Start Trails the default mode, it will boot in video.1

**Reply by bthitney** [reply]
Thanks! Ok I think that’s the lightbulb moment. The order here matters (for labs) and I think I was treating it more like configuration flags and not a series of instructions, if that makes any sense. I appreciate it and will give it a go asap. I probably will try it with the end time also because I’ll have it hooked up to power, but understand the concept.
